views/docs/api.php

```diff
@@ -26,8 +26,8 @@
   <h3>Parameters</h3>
 
   <ul>
-    <li><b>action</b>: Set the action of the form to this service (<code><?= Config::$base ?>auth</code>) or <a href="https://github.com/aaronpk/IndieLogin.com">download the source</a> and run your own server.</li>
-    <li><b>me</b>: (optional) The "me" parameter is the URL that the user enters. If you leave this out, then this website will prompt the user to enter their URL.</li>
+    <li><b>action</b>: Set the action of the form to this service (<code><?= get_setting('base') ?>auth</code>) or <a href="https://github.com/aaronpk/IndieLogin.com">download the source</a> and run your own server.</li>
+    <li><b>me</b>: The "me" parameter is the URL that the user enters.</li>
```

Comment:
Looks like we lost the change here. Can you pull this change in from the current master branch? Might be easiest to just copy in the changes again.

Reply:
Yeah, sorry about that @aaronpk I've just re-pushed, but I'll look elsewhere with w=1 to see if anything else jumps out